.github/workflows/build.yaml

```diff
@@ -9,11 +9,39 @@ on:
       - master
 
 jobs:
-  hello:
-    name: Hello
+  static-analysis:
+    name: Static Analysis
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
```

Comment:
If we decide to run the static analysis on PHP 7.4, we only need to change the value here.